uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -214,6 +218,7 @@
 ############### Run runTarchiveLoader###########################
 ################################################################
 $output = $imaging_upload->runTarchiveLoader();
+$imaging_upload->updateMRIUploadTable('Inserting', 0);
```

Comment:
This is actually the imaging_upload_file.pl (not tarchiveLoader). Line 220 of this script takes you eventually to the tarchiveLoader, you rurn it, you return to this script, then on line 221 we set Inserting back to '0' (as tarchiveLoader is the last step of the insertion scripts).